<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DashboardPage</class>
 <widget class="QWidget" name="DashboardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dashboard</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="dashboardTitleLabel">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Dashboard Overview</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="statsGrid">
     <property name="spacing">
      <number>15</number>
     </property>
     <item row="0" column="0">
      <widget class="QWidget" name="cardTotalProducts" native="true">
       <property name="styleSheet">
        <string notr="true">#cardTotalProducts {
border: 1px solid #e0e0e0;
border-radius: 5px;
background-color: #ffffff;
}</string>
       </property>
       <layout class="QVBoxLayout" name="cardTotalProductsLayout">
        <item>
         <widget class="QLabel" name="totalProductsTitleLabel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Total Products</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="totalProductsValueLabel">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="cardTotalItems" native="true">
       <property name="styleSheet">
        <string notr="true">#cardTotalItems {
border: 1px solid #e0e0e0;
border-radius: 5px;
background-color: #ffffff;
}</string>
       </property>
       <layout class="QVBoxLayout" name="cardTotalItemsLayout">
        <item>
         <widget class="QLabel" name="totalItemsTitleLabel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Total Items in Stock</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="totalItemsValueLabel">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QWidget" name="cardStockValue" native="true">
       <property name="styleSheet">
        <string notr="true">#cardStockValue {
border: 1px solid #e0e0e0;
border-radius: 5px;
background-color: #ffffff;
}</string>
       </property>
       <layout class="QVBoxLayout" name="cardStockValueLayout">
        <item>
         <widget class="QLabel" name="stockValueTitleLabel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Total Stock Value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="stockValueValueLabel">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0.00 MAD</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QWidget" name="cardTotalRevenue" native="true">
       <property name="styleSheet">
        <string notr="true">#cardTotalRevenue {
border: 1px solid #e0e0e0;
border-radius: 5px;
background-color: #ffffff;
}</string>
       </property>
       <layout class="QVBoxLayout" name="cardTotalRevenueLayout">
        <item>
         <widget class="QLabel" name="totalRevenueTitleLabel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Total Revenue</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="totalRevenueValueLabel">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0.00 MAD</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QWidget" name="cardSalesToday" native="true">
       <property name="styleSheet">
        <string notr="true">#cardSalesToday {
border: 1px solid #e0e0e0;
border-radius: 5px;
background-color: #ffffff;
}</string>
       </property>
       <layout class="QVBoxLayout" name="cardSalesTodayLayout">
        <item>
         <widget class="QLabel" name="salesTodayTitleLabel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Sales Today</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="salesTodayValueLabel">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QWidget" name="cardSalesMonth" native="true">
       <property name="styleSheet">
        <string notr="true">#cardSalesMonth {
border: 1px solid #e0e0e0;
border-radius: 5px;
background-color: #ffffff;
}</string>
       </property>
       <layout class="QVBoxLayout" name="cardSalesMonthLayout">
        <item>
         <widget class="QLabel" name="salesMonthTitleLabel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Sales This Month</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="salesMonthValueLabel">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="weeklySalesCard" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="weeklySalesTitle">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Sales for Last 7 Days</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="weeklySalesTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
